src/components/media/MediasSelectModal.jsx

```diff
@@ -89,7 +89,7 @@ const MediasSelectModal = ({
           <p>
             For {mediaRoom} other than
             {mediaRoom === "images"
-              ? ` 'png', 'jpg', 'gif', 'tif', 'bmp', 'ico', 'svg'`
+              ? ` 'png', 'jpg', 'gif', 'tif', 'bmp', 'ico', 'svg', 'webp'`
```

Comment:
similarly for this and other occurrences down below

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/1180/commits/1fbc5c845d558299b123ec0568659337e6a15d8f